machine.go

```diff
@@ -148,8 +148,12 @@ type Machine struct {
 	cmd           *exec.Cmd
 	logger        *log.Entry
 	machineConfig models.MachineConfiguration // The actual machine config as reported by Firecracker
-	errCh         chan error
-	startOnce     sync.Once
+	// startOnce ensures that the machine can only be started once
+	startOnce sync.Once
+	// startCh is a channel which gets closed when the VMM is started
```

Comment:
exitCh?

Reply:
Oops, deleted the wrong lines.  startCh was in an earlier draft of this commit, but I removed it as it was no longer useful.

Reply:
Fixed.